src/main/java/duke/core/Duke.java

```diff
@@ -0,0 +1,56 @@
+package duke.core;
+
+import duke.command.Command;
+import java.io.File;
```

Comment:
Should the import statements be more consistent in format? I.e. New line between duke imports and standard java imports, just like in the other files.